modules/features/falcon_feature_commerce_direct_debit_gateway/falcon_feature_commerce_direct_debit_gateway.info.yml

```diff
@@ -3,7 +3,9 @@ description: 'Direct Debit gateways.'
 type: module
 core: 8.x
 dependencies:
-  - commerce_payment
   - commerce_directdebit
+  - commerce_payment
+  - config_snapshot
+  - falcon_feature_commerce_direct_debit_gateway
```

Comment:
Could you please restore the dependencies?

Reply:
@kalabro done